coins/src/adapters/tokenMapping_added.json

```diff
@@ -8031,6 +8031,12 @@
       "decimals": "18",
       "symbol": "SolvBTC.m",
       "to": "coingecko#solv-btc"
+    },
+    "0xe85411c030fb32a9d8b14bbbc6cb19417391f711": {
```

Comment:
Instead add this to `/tokenMapping.json`. There is no need for the changes in the other 2 files. code in /bridges/ is generally for dynamic mapping creation which isnt necessary in this case.